### DISCUSSION TITLE
Feature Request - Ldap bind user

### DISCUSSION THREAD

**Discussion by PotentialIngenuity** [mealie-recipes/mealie#1585] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
The ldap feature was implemented with authless searches. It would be nice if we could specify a bind user to connect with. Many ldap directories require authentication to lookup users.

**(Optional): Suggest A Solution**

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by Elegant996** [comment]
This would require a constant polling of the LDAP users. The current behavior is to look an indvidual LDAP user and create them in mealie if they are able to authenticate against the LDAP server. You would need to constantly poll to keep info up to date.

I thought about implementing this myself but have opted to just make minor improvements on the current implementation for the moment.

**Comment by Elegant996** [comment]
@hay-kot Reviewed the PR again, should be good to go. Not sure how to prevent the error regarding security.py from generating. Thanks!
